etl/search.py

```diff
@@ -22,4 +28,19 @@ def create_metrics_search_js(metrics, legacy=False):
         if metric_val.get("expires") == "never":
             del metric_val["expires"]
 
+    if app_name == "fog":
+        # since the FOG data will be consolidated with legacy data as one API
+        # we add a "fog" property to each metric to differentiate between
+        # FOG and legacy probes, for easier visual treatment implementation in GLAM
+        for metric_val in metric_data.values():
+            metric_val["fog"] = True
+        return FOG_DATA_JS_TEMPLATE.render(
+            metric_data=dump_json(metric_data), legacy=dump_json(legacy)
+        )
```

Comment:
What if the user is searching for FOG only data? Wouldn't this break that use case? IIUC we are not creating a search template for that case.